src/components/Button/BackButton.js

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import { IoIosArrowBack } from "react-icons/io";
+import { withRouter } from 'react-router-dom';
+
+const BackButton = ({history}) => {
+  return (
+    <div
+    onClick={() => history.goBack()}
```

Comment:
Please switch to HTML buttons for accessibility

Reply:
Fixed